src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -58,6 +58,9 @@
     <ExpectedDependency Include="System.Runtime.CompilerServices.Unsafe" OptimizeAssemblies="lib/netstandard2.0/System.Runtime.CompilerServices.Unsafe.dll" />
     <ExpectedDependency Include="System.Text.Encoding.CodePages" OptimizeAssemblies="lib/netstandard2.0/System.Text.Encoding.CodePages.dll" />
     <ExpectedDependency Include="System.Numerics.Vectors"/>
+    <ExpectedDependency Include="Nerdbank.Streams"/>
+    <ExpectedDependency Include="System.IO.Pipelines"/>
+    <ExpectedDependency Include="System.ValueTuple"/>
```

Comment:
>System.ValueTuple [](start = 33, length = 17)

System.ValueTuple should not be inserted. Something's not right with the packages we reference.

Reply:
@AArnott do you know why depending on StreamJsonRpc 2.0 require these new dependency?

Reply:
Mostly yes.
1. Pipelines gives us higher throughput because it separates transmission from serialization of messages. 
1. ValueTuple... I don't remember. 
1. Nerdbank.Streams: fills in a bunch of functionality gaps that corefx does not yet offer around the `IBufferWriter<T>` interface and interop with streams, etc.

All these assemblies are already included in VS.

Reply:
BTW: Nerdbank.Streams also is where our multiplexing stream feature is. So with this dependency you can start sending multiple channels of arbitrarily mixed protocols across the same (ServiceHub) Stream.

Reply:
@AArnott yep. I already looked into it. I will start using it once this move get checked in.